vcd/datasource_vcd_lb_app_rule.go

```diff
@@ -0,0 +1,61 @@
+package vcd
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func datasourceVcdLBAppRule() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceVcdLBAppRuleRead,
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "vCD organization in which the LB Application Rule is located",
+			},
+			"vdc": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "vCD virtual datacenter in which the LB Application Rule is located",
+			},
+			"edge_gateway": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Edge gateway name in which the LB Application Rule is located",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "LB Application Rule name for lookup",
+			},
+			"script": {
+				Computed: true,
+				Type:     schema.TypeString,
+				Description: "The script for the LB Application Rule. Each line will be separated " +
```

Comment:
technically, one line is "_terminated_ by a newline", while "many lines are _separated_ by newlines".
I prefer using "newline" instead of "\n character".

Reply:
Mentioned both. Hopefully this makes it clearer.